src/main/java/frc/robot/commands/Shoot.java

```diff
@@ -14,7 +14,7 @@ public Shoot(ShooterSubsystem shooterSubsystem , double power) {
 
     @Override
     public void execute() {
-        shooter.rampUpShooter(power);
+        shooter.setRPM(power);
```

Comment:
Since we now use instant commands for setRPM, can we deprecate this? Not sure what use case we have for this now

Reply:
We can deprecate this.